app/services/file_generator.rb

```diff
@@ -79,12 +79,12 @@ def blob
   end
 
   sig { returns(Hash) }
-  # TODO: This varies based on what the user selected
   def access
-    {
-      access: 'stanford',
-      download: 'stanford'
-    }
+    if attached_file.hide?
+      { access: 'dark', download: 'none' }
+    else
+      { access: 'world', download: 'world' }
```

Comment:
Should this vary based on the access specified for the work? Is it OK if the work is set to Stanford-only and its files are world?

Reply:
@mjgiarlo currently Fedora 3 doesn't have a way to persist this, so it's either "world" (which defers to the object level access) or  "dark" which marks "publish=no".

Reply:
OK.